### PR TITLE
Prep v2.33.0 Release

### PR DESCRIPTION
**Pull request by danaelhe**

### CODE REVIEW

**Review by andrewsomething — CHANGES_REQUESTED** [review]
There are some issues with  #1078 that we should address before releasing this.

It adds `port` to the schema, but it is never actually used. We need to also add it in the `expandAppHealthCheck` and `flattenAppHealthCheck` functions or it will not effect anything.

**Comment by danaelhe** [comment]
> There are some issues with #1078 that we should address before releasing this.
> 
> It adds `port` to the schema, but it is never actually used. We need to also add it in the `expandAppHealthCheck` and `flattenAppHealthCheck` functions or it will not effect anything.

Oh yikes, good catch. I'll push a PR now.

**Comment by nemcikjan** [comment]
@danaelhe should I fix it?

**Comment by danaelhe** [comment]
> @danaelhe should I fix it?

Thank you but it's already been fixed in #1080 👍

**Comment by andrewsomething** [comment]
FYI, I also reviewed and merged https://github.com/digitalocean/terraform-provider-digitalocean/pull/1074 so it could make it into the release.

**Comment by danaelhe** [comment]
> FYI, I also reviewed and merged #1074 so it could make it into the release.

updated change log in latest commit 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+- #1073 - @T-jegou - Add `digitalocean_database_connection_pool` datasource
+
+BUG FIXES:
+
+- #1078 - @nemcikjan - fix: added missing option to set port on health_check
+- #1076 - @dweinshenker - Remove unclean_leader_election_enable for kafka topic configuration
+- #1080 - @danaelhe - Apps: Reference Port in expandAppHealthCheck and flattenAppHealthCheck
+- #1074 - @T-jegou - Fixing Case Handling for Volume Resource
+
+MISC:
```
Comment (andrewsomething) [inline CHANGELOG.md:14]:
```suggestion
```

**Review by andrewsomething — APPROVED** [review]
:rocket:

**Pull request merged**